lib/utils/import.js

```diff
@@ -49,3 +50,32 @@ function getSourceModuleName(node) {
     return undefined;
   }
 }
+
+/**
+ * Gets an import identifier (either imported or local name) from the specified ImportDeclaration.
+ *
+ * @param {node} node the ImportDeclaration to find the import identifier for
+ * @param {*} source the source, or module name string, of the import
+ * @param {*} importName the imported name to find (will return the alias of the import, of found)
+ * @returns
+ */
+function getImportIdentifier(node, source, importName) {
+  assert(
+    isImportDeclaration(node),
+    `getImportIdentifier should be called with a node that's type is 'ImportDeclaration'. You passed '${node.type}'`
+  );
+
+  if (node.source.value !== source) {
+    return '';
```

Comment:
We should return `null` or `undefined` if there's no match.

Reply:
Agree. Updated to maintain consistent return types.